src/Common/ZooKeeper/IKeeper.h

```diff
@@ -510,6 +510,18 @@ class Exception : public DB::Exception
     const Error code;
 };
 
+class SimpleFaultInjection
+{
+public:
+    SimpleFaultInjection(Float64 probability_before, Float64 probability_after_, const String & description_);
+    ~SimpleFaultInjection() noexcept(false);
```

Comment:
My c++ knowledge is bad... What does it mean `noexpect` with argument in method declaration?

Reply:
Dtors are noexcept by default, `noexcept(false)` disables that